package.json

```diff
@@ -49,7 +49,7 @@
   "engines": {
     "vscode": "^1.43.0"
   },
-  "enableProposedApi": true,
+  "enableProposedApi": false,
```

Comment:
Why this change? I thought we set it true so beta builds work with proposed API, then the release action changes it to false. I know this property is going away at 1.63, but at the moment we declare 1.43 as our minimum version. And 1.63 hasn't shipped yet.

Reply:
yeah, returned it back